### PR TITLE
Cache strategy updates

### PR DESCRIPTION
**Pull request by lxdvs**

### CODE REVIEW

**Comment by nwadams** [comment]
lgtm. I use this by calling setShouldRetrieveCache(false) right?

**Comment by lxdvs** [comment]
yep

File: src/com/android/volley/RequestQueue.java [hunk 1]
```diff
@@ -263,7 +263,7 @@ public Request add(Request request) {
         request.addMarker("add-to-queue");
 
         // If the request is uncacheable, skip the cache queue and go straight to the network.
-        if (!request.shouldCache()) {
+        if (!request.shouldUseCache()) {
```
Comment (carolineleung) [inline src/com/android/volley/RequestQueue.java:266]: +1

**Comment by carolineleung** [comment]
LGTM

**Pull request closed without merging**